manual_refresh_token_provider.go

```diff
@@ -0,0 +1,33 @@
+package incognia
+
+import "errors"
+
+var ErrNoToken = errors.New("no token stored in memory")
```

Comment:
Always use parenthesis with `import` and top-level `var` and `const` blocks.

```suggestion
import (
	"errors"
)

var (
	ErrNoToken = errors.New("no token stored in memory")
)
```